src/app/components/cds/media-cards/MediaCount.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import ButtonMain from '../../cds/buttons-checkboxes-chips/ButtonMain';
+import Tooltip from '../../cds/alerts-and-prompts/Tooltip';
+import MediaIcon from '../../../icons/perm_media.svg';
+import MediaTypeDisplayIcon from '../../media/MediaTypeDisplayIcon';
+import { getCompactNumber, getSeparatedNumber } from '../../../helpers';
+
+const MediaCount = ({
+  intl,
+  mediaCount,
+  mediaType,
+}) => (
+  <FormattedMessage id="sharedItemCard.medias" defaultMessage="Medias" description="This appears as a label next to a number, like '1,234 Medias'. It should indicate to the user that whatever number they are viewing represents the number of medias attached to an item .">
+    { mediasLabel => (
+      <Tooltip
+        arrow
+        title={`${getSeparatedNumber(intl.locale, mediaCount)} ${mediasLabel}`}
```

Comment:
I think it would be more appropriate to make the localizable string pluralized to account for 0, 1..N medias and also for better translation. We can't be sure that all languages will be translated number then "medias" in the same order.